_includes/variables.liquid

```diff
@@ -0,0 +1,82 @@
+<!-- Site-wide variables -->
+{% if page.lang == nil or page.lang == "en" %}
+  <!-- Page language -->
+  {% assign lang = "" %}
+  <!-- Includes -->
+  {% assign includes = site.data.includes %}
+  <!-- Architecture -->
+  {% assign architecture = site.data.architecture %}
+  <!-- Doc index -->
+  {% assign doc-index = site.data.doc-index %}
+  <!-- Downloads page -->
+  {% assign downloads-page = site.data.downloads-page %}
+  <!-- HCL -->
+  {% assign hcl = site.data.hcl %}
+  <!-- Experts -->
+  {% assign experts = site.data.experts %}
+  <!-- Team page -->
+  {% assign team-page = site.data.team-page %}
+  <!-- Style guide page -->
+  {% assign style-guide-page = site.data.style-guide-page %}
+{% else %}
+  <!-- Page language -->
+  {% assign lang = "/" | append: page.lang %}
+  <!-- Includes -->
+  {% assign includes = site.data.translation[page.lang].includes %}
+  {% if includes == nil or includes.size == 0 %}
+    {% assign includes = site.data.includes %}
+  {% endif %}
+  <!-- Architecture -->
+  {% assign architecture = site.data.translation[page.lang].architecture %}
+  {% if architecture == nil or architecture.size == 0 %}
+    {% assign architecture = site.data.architecture %}
+  {% endif %}
+  <!-- Doc index -->
+  {% assign doc-index = site.data.translation[page.lang].doc-index %}
+  {% if doc-index == nil or doc-index.size == 0 %}
+    {% assign doc-index = site.data.doc-index %}
+  {% endif %}
+  <!-- Downloads page -->
+  {% assign downloads-page = site.data.translation[page.lang].downloads-page %}
+  {% if downloads-page == nil or downloads-page.size == 0 %}
+    {% assign downloads-page = site.data.downloads-page %}
+  {% endif %}
+  <!-- HCL -->
+  {% assign hcl = site.data.translation[page.lang].hcl %}
+  {% if hcl == nil or hcl.size == 0 %}
+    {% assign hcl = site.data.hcl %}
+  {% endif %}
+  <!-- Experts -->
+  {% assign experts = site.data.translation[page.lang].experts %}
+  {% if experts == nil or experts.size == 0 %}
+    {% assign experts = site.data.experts %}
+  {% endif %}
+  <!-- Team page -->
+  {% assign team-page = site.data.translation[page.lang].team-page %}
+  {% if team-page == nil or team-page.size == 0 %}
+    {% assign team-page = site.data.team-page %}
+  {% endif %}
+  <!-- Style guide page -->
+  {% assign style-guide-page = site.data.translation[page.lang].style-guide-page %}
+  {% if style-guide-page == nil or style-guide-page.size == 0 %}
+    {% assign style-guide-page = site.data.hcl %}
+  {% endif %}
+{% endif %}
+<!-- Doc link -->
+{% assign doc_link = lang | append: "/doc/" %}
+{% assign doc_count = site.doc | concat: site.qubes-translated | where:'permalink', doc_link %}
+{% if doc_count.size == 0 %}
+  {% assign doc_link = "/doc/" %}
+{% endif %}
+<!-- Team link -->
+{% assign team_link = lang | append: "/team/" %}
+{% assign team_count = site.pages | concat: site.qubes-translated | where:'permalink', team_link %}
```

Comment:
For this one again, searching just `site.qubes-translated` should be enough. But maybe also `site.data.translation[page.lang].team`? Or maybe even set it together with `team-page` variable above?

Reply:
(Repeat comment, also applies here:)

I actually have no idea. I simply copied this line verbatim from the existing code (i.e., abstracted it out without modifying it).

I will try to implement your suggestion, but I'm not set up to test the translated website.

Reply:
Done: https://github.com/QubesOS/qubesos.github.io/pull/178/commits/59a5d15deb5813683510995ec58ffa43393d5123

Is this what you had in mind?

Reply:
Yes, exactly.